### DISCUSSION TITLE
Alarmierungen zusammenführen

### DISCUSSION THREAD

**Discussion by Cryjack** [Schrolli91/BOSWatch#542] [General]
Hallo Gemeinde,

ich möchte gerne folgendes für ein Plugin erreichen:

- Ich erhalte eine Alarmierung, diese soll ganz normal verarbeitet werden.
- Eine weitere Alarmierung auf der selben RIC folgt mit ergänzenden Informationen (Fahrzeuge)
- Eine weitere Alarmierung auf einer weiteren RIC folgt mit Geokoordinaten

Ich möchte diese Informationen (in einem gewissen Zeitraum empfangen) zusammenführen um somit die Alarmierung zu "erweitern". Dazu nimmt die Schnittstelle des Empängers eine ID entgegen, welche ich in weiterer Verwendung einfach mit neuen Datenfeldern beschreiben kann. Gibt es eine Möglichkeit, eine ID über diese Alarme im Plugin zu speichern um diese dann bei den weiteren Alarmierungen zu nutzen?
Im Prinzip so ähnlich wie die Erkennung der doppelten Alarme.

Danke für die Unterstützung.

**Comment by Cryjack** [comment]
Kann ich im Plugin eine Variable speichern und beim erneuten Aufruf abfragen?

**Comment by Schrolli91** [comment]
> Kann ich im Plugin eine Variable speichern und beim erneuten Aufruf abfragen?

Sollte im Prinzip gehen wie im doubleFilter. Hast du ja schon selbst richtig erkannt.
Kommen diese 3 Alarmierungen denn direkt nacheinander, oder können da auch andere RICS dazwischen funken?
